moda-contracts/src/ModaRegistry.sol

```diff
@@ -1,14 +1,24 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.21;
 
-import "./interfaces/IModaRegistry.sol";
-import "./interfaces/IMembership.sol";
+import {IModaRegistry} from "./interfaces/IModaRegistry.sol";
+import {IMembership} from "./interfaces/IMembership.sol";
 import "@openzeppelin/contracts/access/extensions/AccessControlEnumerable.sol";
 
 contract ModaRegistry is IModaRegistry, AccessControlEnumerable {
     using EnumerableSet for EnumerableSet.AddressSet;
 
     /// State Variables
+
+    /// @dev only an address with a verifier role can verify a track
+    bytes32 public constant VERIFIER_ROLE = keccak256("VERIFIER_ROLE");
+
+    /// @dev only an address with a releases registrar role can register a releases contract
+    bytes32 public constant RELEASES_REGISTRAR_ROLE = keccak256("RELEASES_REGISTRAR_ROLE");
+
+    /// @dev an address with a gold role will have their tracks verified on registration
+    bytes32 public constant GOLD_ROLE = keccak256("GOLD_ROLE");
```

Comment:
I think we should call this `AUTO_VERIFIED_ROLE`

Reply:
done